### PR TITLE
Embed polestar

### PR DESCRIPTION
**Pull request by domoritz**
https://github.com/uwdata/vega-lite-ui/pull/61

Also embeds fonts and images in compiled css.

### CODE REVIEW

**Comment by domoritz** [comment]
Fixes https://github.com/uwdata/ipython-vega-lite/issues/1

**Comment by domoritz** [comment]
Fixes https://github.com/uwdata/polestar/issues/311

File: gulp/build.js [hunk 1]
```diff
@@ -46,7 +46,11 @@ gulp.task('html', ['inject', 'partials'], function () {
     .pipe($.uglify({preserveComments: $.uglifySaveLicense}))
     .pipe(jsFilter.restore())
     .pipe(cssFilter)
-    .pipe($.replace('../../bower_components/fontawesome/fonts', '../fonts'))
+    //.pipe($.replace('../../bower_components/fontawesome/fonts', '../fonts'))
+    .pipe($.base64({
```
Comment (kanitw) [inline gulp/build.js]: what is this for?

Comment (domoritz) [inline gulp/build.js]: For iPython, we cannot load pictures or fonts, so I have to embed them with base64 encoding into the CSS. It actually reduces the number of requests, which is nice, but increases the overall data size. 

I was too lazy to add a build flag to build different versions.

**Comment by domoritz** [comment]
Want to merge @kanitw?

**Pull request merged**